Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -242,6 +283,18 @@ class EdgeHitProcessor: HitProcessing {
         return requestHeaders
     }
 
+    /// Extracts the config the event data which contains datastreamIdOverride and datastreamConfigOverride details.
+    /// - Parameter event: current event for which the config is to be extracted
+    /// - Returns: the dictionary of extracted config details
+    private func getEventConfig(from event: Event) -> [String: Any]? {
```

Comment:
we can move this method in Event+Edge so you can call it on the event instance `event.config`